src/main/jbake/content/docs/dev/terms.adoc

```diff
@@ -10,15 +10,53 @@ Heiko Rupp
 
 Hawkular sometimes uses terms that are not everyone is familiar with. This page is an effort to clarify those.
 
+== General terms
+
 Dampening:: This is a concept in Hawkular-Alerts that allows to "verify" an alerting condition. As an example,
-sometimes one only wants to trigger an alert when the same conditions occurs at least three times in 5 minutes.
+ sometimes one only wants to trigger an alert when the same conditions occurs at least three times in 5 minutes.
+
 Feed:: A feed is a source of incoming data to pe processed by Hawkular. Usually a feed is something like an agent
-that is collecting data, but it could also be an application sending data or a shell-script using the `curl` command
-to submit the data.
+ that is collecting data, but it could also be an application sending data or a shell-script using the `curl` command
+ to submit the data.
+
 Kettle:: The kettle is a runtime container of Hawkular that contains all the individual Hawkular components including
  Hawkular-Metrics, Hawkular-Alerts, the Pinger and many more
-Pinger:: A component that "pings" footnote::[Not a ICMP-Ping, but a http call.] remote http servers and reports back
+
+Member:: an User or Organization that can be part of an Organization.
+
+Organization:: a non-user which has an Owner and a set of Members. In most cases, consumers of Hawkular Accounts
+ do not need to deal directly with organizations.
+
+Owner:: owns a Resource footnoteref:[resource,A Resource following the definition of Hawkular Accounts], and can be
+(currently) either a Hawkular User or an Organization.
+
+Persona:: either a User or an Organization. A real User can be impersonating an Organization, so, this Organization
+ would be the current Persona. In most cases, the current Persona corresponds to an intuitive notion of the
+ "current user" or "current tenant".
+
+Pinger:: A component that "pings" footnoteref:[ping, Not an ICMP-Ping, but a http call.] remote http servers and
+reports
+back
  the server status along with the time it took.
+
 Trigger-Definition:: The Trigger definition consits of the rules when incoming conditions should trigger (fire) an
-alert.
+ alert.
+
+User:: a real user of the application. In most cases, consumers of Hawkular Accounts do not need to deal directly
```

Comment:
Would it make sense in sentences like this, to have the H-Accounts specific part in the H-Accounts page and in this terms page to only have the more general definitions?